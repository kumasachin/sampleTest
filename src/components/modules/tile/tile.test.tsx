import React from 'react';
import Tile from './tile';
import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() })

it('renders correctly enzyme', () => {
    const wrapper = shallow(<Tile
        clickHandler={()=>{}}
        pendingBalance={132}
        currency={"GBP"}
        id={"1"}
        fundsHeld={123}
        overDraft={123}
        accountDetails= {{
            sortCode: "1231231",
            accountNumber: 111111111,
            id: "123123"
        }}
    />)

    expect(toJson(wrapper)).toMatchSnapshot();
});