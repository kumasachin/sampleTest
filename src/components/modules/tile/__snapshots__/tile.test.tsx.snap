// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<div>
  <WithStyles(ForwardRef(Card))
    className="makeStyles-root-1"
  >
    <WithStyles(ForwardRef(CardHeader))
      title={
        <WithStyles(ForwardRef(Typography))
          component="h2"
          variant="h5"
        >
          A/C Number: 111111111 
        </WithStyles(ForwardRef(Typography))>
      }
    />
    <WithStyles(ForwardRef(CardHeader))
      title={
        <WithStyles(ForwardRef(Typography))
          component="h2"
          variant="h5"
        >
          Balance:  GBP - 132
        </WithStyles(ForwardRef(Typography))>
      }
    />
    <WithStyles(ForwardRef(CardContent))
      className="makeStyles-content-2"
    >
      <WithStyles(ForwardRef(Button))
        name="1"
        onClick={[Function]}
      >
        check details
      </WithStyles(ForwardRef(Button))>
    </WithStyles(ForwardRef(CardContent))>
  </WithStyles(ForwardRef(Card))>
</div>
`;
